app/screens/Explore.js

```diff
@@ -599,8 +599,6 @@ class Explore extends React.Component {
                 : (
                   <StyledMap
                     styleURL={styleURL}
-                    showUserLocation
```

Comment:
These are no longer required.  These settings are now defined in the Camera component.